corgi/tasks/prod_defs.py

```diff
@@ -130,7 +130,11 @@ def update_products() -> None:
                             "obj": product_stream,
                         },
                     )
-
+                    # Some Product Versions in Errata Tool list too many brew tags
+                    # Linking these Product Versions here leads to builds from later streams
+                    # being included in earlier ones, see CORGI-546.
+                    if product_version.name in ["quay-3", "rhscl-3", "dts-11", "dts-12", "rhn_satellite_6"]:
+                        continue
                     if len(brew_tags) > 0:
```

Comment:
After your explanation, I agree it's OK to skip creating the variant from the Brew tag, since we only need to link the Brew tag to a stream and not a variant.

But the check here will also skip creating any variants from the errata_info stanza in prod-defs for these streams (line 189 below). Should we move the check a little lower? Or should we try to get the data in ET updated, so that these PVs don't have incorrect Brew tags?

I might be confused and we don't want any variants here, either from Brew tags or from errata_info. In that case feel free to ignore.
```suggestion
                    # Some Product Versions in Errata Tool list too many brew tags
                    # Linking these Product Versions here leads to builds from later streams
                    # being included in earlier ones, see CORGI-546.
                    if len(brew_tags) > 0 and product_version.name not in ("quay-3", "rhscl-3", "dts-11", "dts-12", "rhn_satellite_6"):
```